docs/group-applications-workflow.md

```diff
@@ -0,0 +1,37 @@
+1. Admin creates an event with slug. Admin remembers the slug.
+
+1. Admin bulk uploads people sent by the partner and assigns them to the training. This requires to use CSV file with five columns:
+
+    - personal name,
+    - family name,
+    - email address,
+    - the training slug (all rows with the same value) and
+    - role ('learner' in all rows).
+
+1. Admin goes to "trainees" page and filter by training. That way, Admin can list exactly the same set of people that he uploaded. In the same view, Admin can filter by whether an training request was sent or not.
+
+1. Admin filters only trainees who haven't sent training request yet (and are assigned to the training) and bulk email them. This email contains link to training request form, with group name. Training id is not included in the link, because trainees are already matched with the training.
```

Comment:
Yes, this option can be available all the time. There are three kinds of email that you may want to send (empty one, asking for confirmation and asking for filling the form). So we can have a button like this:
![image](https://cloud.githubusercontent.com/assets/1645996/17174468/054ea212-5402-11e6-9366-ff1efd27587f.png)
When you click it, an empty email is created. When you click the arrow, you can access other options:
![image](https://cloud.githubusercontent.com/assets/1645996/17174437/c50c16ee-5401-11e6-9f71-e90b9a04ee80.png)

Or, when you click blue button, what happens could be dynamic and depend on the filters:

- If you filtered by training and filtered only people who have sent a request, "Ask to confirm interest in training" option is used. 
- If you filtered by training and filtered only people who have not sent a request, "Ask to submit group application" option is used.
- Otherwise, an empty email is created.

Any opinions?